fuzz/mayhemfiles/Mayhemfile_generate_url

```diff
@@ -0,0 +1,7 @@
+project: path-tree
```

Comment:
This isn't quite the correct formate for Mayhemfiles

Reply:
I changed the format as requested